src/app/visual/phone-field/phone-field-visual.component.html

```diff
@@ -10,6 +10,7 @@
   [@.disabled]="true"
 >
   <sky-phone-field
+    [allowExtensions]='false'
```

Comment:
This should be double-quotes instead of single.

Reply:
Fixed